admin_manual/ai/app_context_chat.rst

```diff
@@ -11,7 +11,7 @@ Context Chat is an :ref:`assistant<ai-app-assistant>` feature that is implemente
 
 Together they provide the ContextChat text processing tasks accessible via the :ref:`Nextcloud Assistant app<ai-app-assistant>`.
 
-The *context_chat* and *context_chat_backend* apps run only open source models and do so entirely on-premises. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
+The *context_chat* and *context_chat_backend* apps can run open source models entirely on-premises or use Free text to text task processing providers like OpenAI integration, LLM2, etc. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
 
 This app supports input and output in languages other than English if the language model supports the language.
```

Comment:
Should we mention that a Free prompt provider is needed now?

Reply:
ah yeah, good point